src/QuantumToolbox.jl

```diff
@@ -49,6 +49,7 @@ include("qobj/eigsolve.jl")
 include("qobj/functions.jl")
 include("qobj/states.jl")
 include("qobj/operators.jl")
+include("qobj/tensor_functions.jl")
```

Comment:
We are following the code structure in `qutip`.

Since the `permute` function is an attribute function in `Qobj` class (in Python), so we put it in `qobj/arithmetic_and_attributes.jl`.